### PR TITLE
[Demo] Add last changes to visual-vocabulary before publishing

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description
- Update dashboard config
- Update and consolidate README in `vizro/README.md` and` vizro/vizro-core/README.md`

## Screenshot

visual_vocabulary

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,9 +102,21 @@ See the [Vizro-AI documentation](https://vizro.readthedocs.io/projects/vizro-ai/
...
+</p>
+
+## Vizro visual vocabulary
+
+Our visual vocabulary dashboard is a guide for selecting and creating various types of charts. It provides guidance
+on when to use each chart type, along with sample Python code using [Plotly](https://plotly.com/python/).
+
+<p align="center">
+<a href="https://huggingface.co/spaces/vizro/demo-visual-vocabulary/">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/visual_vocabulary.png" width="900">
```
Comment (huong-li-nguyen) [inline README.md:118]: Note: The image link will work after merge

File: vizro-core/README.md [hunk 2]
```diff
@@ -90,17 +90,31 @@ Optional high-code extensions allow almost infinite customization in a modular w
...
+</a>
 </p>
 
-### Vizro examples gallery
+## Vizro visual vocabulary
 
-<p align="left">
-<a href="http://vizro.mckinsey.com/">
-<img border="0" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/live_interactive_demo.png" width="525" height="296">  </a>
+Our visual vocabulary dashboard is a guide for selecting and creating various types of charts. It provides guidance
```
Comment (stichbury) [inline vizro-core/README.md:105]:
```suggestion
Our visual vocabulary dashboard helps you to select and create various types of charts. It provides guidance
```

File: vizro-core/examples/_visual-vocabulary/README.md [hunk 3]
```diff
@@ -1,25 +1,35 @@
 # Vizro - Visual vocabulary
```
Comment (stichbury) [inline vizro-core/examples/_visual-vocabulary/README.md:1]:
```suggestion
# Vizro visual vocabulary
```

File: vizro-core/examples/_visual-vocabulary/README.md [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
 
-This dashboard shows a gallery of charts. It includes guidance on when to use each chart type and sample Python code
-to create them using [Plotly](https://plotly.com/python/) and [Vizro](https://github.com/mckinsey/vizro).
+### Welcome to our visual vocabulary dashboard! 🎨
 
-**Created by:**
+This dashboard serves as a comprehensive guide for selecting and creating various types of charts. It provides guidance
```
Comment (stichbury) [inline vizro-core/examples/_visual-vocabulary/README.md:5]:
```suggestion
This dashboard serves as a comprehensive guide for selecting and creating various types of charts. It helps you decide
```

File: vizro-core/examples/_visual-vocabulary/README.md [hunk 5]
```diff
@@ -1,25 +1,35 @@
...
 
 - [Huong Li Nguyen](https://github.com/huong-li-nguyen) and [Antony Milne](https://github.com/antonymilne)
 
 - Images created by QuantumBlack
 
-**Inspired by:**
+Inspired by:
+
+- [The FT Visual Vocabulary](https://github.com/Financial-Times/chart-doctor/blob/main/visual-vocabulary/README.md):
+  Alan Smith, Chris Campbell, Ian Bott, Liz Faunce, Graham Parrish, Billy Ehrenberg, Paul McCallum,Martin Stabe.
```
[the remark this answers is not in the document]
Comment (huong-li-nguyen) [reply vizro-core/examples/_visual-vocabulary/README.md:23]: Hmm.. good question. Only one person has contributed to the actual github repository. The above I've taken from their official poster: https://github.com/Financial-Times/chart-doctor/blob/main/visual-vocabulary/README.md

[the remark this answers is not in the document]
Comment (huong-li-nguyen) [reply vizro-core/examples/_visual-vocabulary/README.md]: I added the Github profiles, where I could find it: https://github.com/mckinsey/vizro/pull/693/commits/8d68355aae6a2afa2de71f0f818b33618859e669

**Review by petar-qb — APPROVED** [review]
This really looks good 👍

**Review by stichbury — APPROVED** [review]
Approving this time 🤦‍♀️

**Pull request merged**